### DISCUSSION TITLE
replacer functionality

### DISCUSSION THREAD

**Discussion by jostendorf** [KyleRoss/node-lambda-log#69] [Q&A]
Are there any examples of how somebody might use the replacer functionality?  We have extracted the logger into a separate typescript file where we are injecting some metadata on every call.  Wondering how I might globally use the replacer functionality to replace PII data.  

**logger.ts:**
export log = new LambdaLog({
dynamicMeta: () => {
  return {
     parameter: "This is fun"
}
}
});

**usage:**
import { log } from 'logger';

log.info('This is a message');

**Comment by KyleRoss** [comment]
You can read more about the replacer here:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#the_replacer_parameter
